test/standalone/Soil/soil_test_3d.jl

```diff
@@ -227,7 +227,7 @@ for FT in (Float32, Float64)
                 K(θX) * dψdθ(θX) * d2θdx2(XX, myZ) +
                 K(θX) * dθdx(XX, myZ)^2.0 * d2ψdθ2(θX)
             )
-            @test maximum(abs.(dYX .- expected)) < 10^10 * eps(FT)
+            @test maximum(abs.(dYX .- expected)) < 10^11 * eps(FT)
```

Comment:
looks fine! The 10^10(11) is arbitrary, but I realized it's not an easy thing to compute a priori -  like we discussed the other day, when making many floating point computations per point in the field, and then taking a statistic of many samples (like a mean), knowing what "floating point error" translates to is hard - but this is even harder, because we are taking a maximum as our statistic, and order statistics depend on the number of samples.